src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -59,7 +59,7 @@ export default class SaveInProgressIntro extends React.Component {
     if (profile.loading && !this.props.resumeOnly) {
       return (
         <div>
-          <LoadingIndicator message="We’re checking to see if you have a saved version of this application"/>
+          <LoadingIndicator message="We’re checking to see if you have a saved version of this application."/>
```

Comment:
Thanks @robertfairhead. Will change the last two. @bethpotts @peggygannon does "Checking to see if you have a saved version of this application..." work for you? If so, I'll make that change as well.